src/common/select-objects.js

```diff
@@ -785,6 +785,10 @@ export class SelectResourceSetsNetwork extends Component {
 
 // ===================================================================
 
+@addSubscriptions(() => ({
```

Comment:
You can also add a comment in `addSubscription` definition explaining why a function can be passed.